api/src/main/resources/application-h2.properties

```diff
@@ -29,3 +29,5 @@ spring.datasource.password=
 spring.datasource.driver-class-name=org.h2.Driver
 
 spring.flyway.locations=classpath:db/h2
+
+slack.url=""
```

Comment:
If this property is set in application.properties, you don't need to include it in other profiles unless you want to change its value